code/__DEFINES/is_helpers.dm

```diff
@@ -77,6 +77,8 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define ishuman(A) (istype(A, /mob/living/carbon/human))
 
+#define ishumantesting(A) (istype(A, /mob/living/carbon/human/consistent) || istype(A, /mob/living/carbon/human/dummy))
```

Comment:
Specifically when we need to run something in a testing enviroment that wouldnt work in a real environment 
(the clockie gear shocks nonclockies for instance when worn)